dockerFileExplain.txt

```diff
@@ -0,0 +1,40 @@
+version : the version of the docker file
+
+services : # the names of the containers 
+
+
+image : the name of the image to pull / that is in the same folder with the docker file that need to be build 
```

Comment:
it is only for images, if you want to build an image with a dockerfile you should use "build"